app/src/main/java/helium314/keyboard/settings/screens/AboutScreen.kt

```diff
@@ -116,14 +118,25 @@ fun createAboutSettings(context: Context) = listOf(
                 val link = ("<a href=\"https://developer.android.com/reference/android/content/Context#createDeviceProtectedStorageContext()\">"
                         + ctx.getString(R.string.hidden_features_text) + "</a>")
                 val message = ctx.getString(R.string.hidden_features_message, link)
-                val dialogMessage = SpannableStringUtils.fromHtml(message)
+val htmlString = message 
+val dialogMessage: Spanned = if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.N) {
+    Html.fromHtml(htmlString, Html.FROM_HTML_MODE_LEGACY)
+} else {
+    @Suppress("DEPRECATION")
+    Html.fromHtml(htmlString)
+}
                 val builder = AlertDialog.Builder(ctx)
                     .setIcon(R.drawable.ic_settings_about_hidden_features)
                     .setTitle(R.string.hidden_features_title)
                     .setMessage(dialogMessage)
                     .setPositiveButton(R.string.dialog_close, null)
                     .create()
                 builder.show()
+                val messageView = builder.findViewById<TextView>(android.R.id.message)
+                if (messageView != null) {
+                    messageView.movementMethod = LinkMovementMethod.getInstance() 
+                    messageView.setTextColor(android.graphics.Color.BLACK)    
```

Comment:
This sets black text in all cases, which makes the text very hard to read in dark mode. You could use the `foreground` color from app resources, that one should work.